### DISCUSSION TITLE
哪時候才能上"大一"CPR?

### DISCUSSION THREAD

**Discussion by cindy5D55** [1101-GS4719-C/Python-Community-GS4719-C#1468] [General]
想問一下大家有人拿到大一CPR的時數了嗎?

問過附近的人發現即使大早上去點擊報名，也沒有聽到成功案例。

**問題**

1.CPR上下學期各開一次，一次500人，所以每年會有1000人可報名，但以111年入學人數來說男975女583，就超過可報人數，再加上大二以上的未報人數遠遠超過1000人。

2.在畢業前的沒拿到時數的補救方案就是加入急救社並參加急救社課程，但也只能如此，即使你有急救執照，好像也無法抵免。

**學校解決較好辦法**

我認為既然大一CPR是中央學生的必修課程，既然如此，可以綁訂服務學習一起上課，或是整個系都排定時間一起上課，畢竟我認為這種”報名”方式比起大一週會可以選自己想要聽的內容，對於CPR來說好像沒有任何意義。

**最後想問一下，大家有甚麼解決的辦法嗎?**

**Comment by Chiuhsianglin** [comment]
大一上的時候我們寢的有一起六點起床就是為了搶CPR，我記得當時學校給的連結點不進去，還是我自己從網站點進去找才順利搶到，也是困難重重，我認為都叫大一CPR了，應該直接幫新生排定時間去上課！

**Comment by Elaine2899** [comment]
我非常認同綁定時間一起去上課這樣的方式，而非大家需要早起搶名額，而且也容易造成名額浪費。經常會有人搶到名額了但排定的時間臨時有事而放棄搶到的名額，這樣的名額浪費可能導致不少人到需要等到大四才搶得到名額。一來一往，不只浪費新生時間更浪費舊生的時間。

**Comment by zephyrxin01** [comment]
我覺得統一時間一起去上課是不錯的解方(像是體育課之類的時間)，畢竟常常有人報名了卻沒有去參加，很浪費名額。但是可能開課單位考量的點是講課的急救員無法在平日的白天配合，如果要在平日晚上開課，可能相對來說比較難統一班上全部人的意見。

**Comment by Aiame** [comment]
我反倒覺得沒必要為了搶CPR早起耶，因為我記得學校都會提供大四優先名額，而且時段是和其他年級分開的，大不了就等大四再去cpr阿，如果沒有想提早畢業的話，給你參考

**Comment by sjh501** [comment]
我覺得大一CPR的初衷很好，但搶不到和抵免限制過多也確實是很讓人詬病。我覺得能夠和一些全校必修的課程放在一起做會不錯，雖然還是會有一些人會因為個人因素沒辦法參與到，但至少大部分都是可以參與到，也不用擔心要早起用搶的，學校網頁也不會因此當機。

**Comment by Alexgns** [comment]
學校感覺不會想要做出一些改變或採納其他建議，反正麻煩的是學生

**Comment by pudding0803** [comment]
畢竟每個人可能都有自己的規劃、安排，讓學生自己選擇時間確實是不錯的做法。但我認為學校應該要多去解決學生的問題，像是為何都知道報名人數會遠大於人數上限，學校卻沒有任何作為？這點似乎跟其他議題都有異曲同工之妙，學生都把想法提供出來了，但行政人員或許是怕麻煩，無論作法多好，都沒有人想要試著實現

**Comment by Tzuhsuannnnn** [comment]
在大一要搶CPR的時候，我超早就起床，網路當時也超當超難搶...但還是十分幸運有搶到，我十分贊同你的想法，省去搶的動作，直接安排在大一課程的確會方便很多，但我認為校方可能有授課人員不足或是時間安排上等其他問題，所以無法執行。

**Comment by Aaron0810** [comment]
我現在是個大三生，之前真的是每學期都有特地爬起來就為了搶這個，結果沒有一次搶到，現在整個時數表就差這5小時，早知道每次都搶不到的話，就乾脆直接等到大四再做就好了，這系統真的蠻白癡的，叫大一CPR應該就要像大一服務學習一樣變成強制做的就好了，不知道幹嘛要弄成這樣用搶的。

**Comment by Superlan8** [comment]
我認為這個制度真的要好好修一修，大一CPR顧名思義就是希望大家在大一就修完，但名額卻總是都要用搶的，導致大家在大一時沒搶到名額，到大二或大三甚至大四時，又去跟下一屆大一搶名額，這產生一個無限的惡性循環，我認為要解決就得從根本做起，在大一就解決這種問題，以免造成惡性循環。

**Comment by Brownnnn78** [comment]
當然是成功的在大一就選到了，我怎麼記得我那時候還很悠哉的在選時間啊，好像看宿舍的大家在問然後就挑好時間了，看到很多人到大四最後一學期才報名到的我，不知道是不是不小心運氣爆棚，

**Comment by juliaouo** [comment]
我當時是定鬧鐘在報名前10分鐘起床很順利就報到了欸？而且當時看還有很多沒報滿，是我太幸運嗎XD報完就回去睡覺了。雖然cpr很難報，但我認為大一cpr如果每個系都安排時間一起上課還是會有人沒辦法去，那這樣那些人怎麼辦？而且現在的報名人數在模擬測驗時就已經要很久了，我認為時間因素是個問題。

**Comment by 111607004** [comment]
我記得到了大四下都會有優先保障名額，如果最後還是沒搶道也不用擔心啦，他應該不會成為卡住畢業的問題。我同學之前有用過一個蠻偏門的方法，就是大家一起搶，就算有去過的人也搶，搶到後再在媒人上線的時間把他退掉，然後叫自己的好友馬上過去補位。雖然有點機車但也不失為一個好方法。

**Comment by bestbanana295** [comment]
如果搶不到大一CPR的話也可以參加急救社每個學期都會舉辦的急救員班喔！如果通過測驗的話不僅可以拿到大一CPR時數也可以取得初級急救員證，不妨可以考慮看看。
